pyproject.toml

```
@@ -23,8 +23,8 @@ textfsm = "*"
 async-timeout = "<4.0"
 aiofiles = "*"
 prompt-toolkit = ">2"
-pyarrow = "10.0.1"
-pandas = "1.4.4"
+pyarrow = "11.0"
+pandas = "^1.5"
```

It's probably better to fix the minor version
```suggestion
pandas = "~1.5"
```